static/docs/use-cases/data-registry.md

```diff
@@ -0,0 +1,177 @@
+# Data Registry
+
+In the [Versioning Tutorial](/doc/tutorials/versioning) we use two ZIP files
+containing parts of a dataset with labeled images of cat and dogs. For
+simplicity, these compressed archives are downloaded with `dvc get` from our
+[dataset registry](https://github.com/iterative/dataset-registry), a <abbr>DVC
+project</abbr> hosted on GitHub.
+
+In this document we'll explain the idea behind **shared data registries**, how
+to easily create one with DVC, and the best ways to version datasets based on
+the same example above (without ZIP files!)
```

Comment:
move ! out of the ( )?

Reply:
Hmmmm no, but it seems I'm missing a period after the closing parenthesis, at least according to https://english.stackexchange.com/questions/63572/exclamation-mark-before-closing-parenthesis. Adding.